src/super_gradients/common/environment/monitoring/network.py

```diff
@@ -0,0 +1,14 @@
+import psutil
+from super_gradients.common.environment.monitoring.utils import bytes_to_megabytes
+
+initial_net_io_counters = psutil.net_io_counters()
+
+
+def get_network_sent_mb() -> float:
+    """Number of MegaBytes sent since import"""
+    return bytes_to_megabytes(psutil.net_io_counters().bytes_sent - initial_net_io_counters.bytes_sent)
```

Comment:
here also, instead of aggregation from import, show the value from the last call.

Reply:
perhaps in the case of network and disk, it will be better to show numbers not between 100 samples (with only 0.1 sec between them), but instead to show numbers between one minute to the previous one (or over x seconds). this will require to add a `reset()`  function of course

Reply:
I changed it in https://github.com/Deci-AI/super-gradients/commit/4f0ffa9124fd052d1aaa90e38677ac4268ba1ecf (I still have some comments to fix)
I chose to do it "per second" to have a constant metric that does not depend on how often we aggregate